components/builder-admin/src/http/handlers.rs

```diff
@@ -1,6 +1,4 @@
 // Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
It's my least favorite line.

![gif-keyboard-13452257350869632017](https://cloud.githubusercontent.com/assets/4304/24878452/0501967e-1de8-11e7-865e-7b5e39ec1207.gif)